### ISSUE TITLE
*Storage does not implement certmagic.Storage

### ISSUE THREAD

**Issue by derevandal** [silinternational/certmagic-storage-dynamodb#9]
Hi, i'm using this plugin and i've got this:

```bash
/home/derevandal/go/pkg/mod/github.com/silinternational/certmagic-storage-dynamodb@v0.0.0-20200506004444-459dfd7e722b/caddy.go:23:2: cannot use s (type *Storage) as type certmagic.Storage in return argument:
        *Storage does not implement certmagic.Storage (wrong type for Lock method)
                have Lock(string) error
                want Lock(context.Context, string) error
```

`main.go`
```go
package main

import (
	"github.com/caddyserver/certmagic"
	caddycmd "github.com/caddyserver/caddy/v2/cmd"

	// plug in Caddy modules here
	_ "github.com/caddyserver/caddy/v2/modules/standard"
	dynamodbstore "github.com/silinternational/certmagic-storage-dynamodb"
	
)

func main() {
	caddycmd.Main()
}

certmagic.Default.Storage = &dynamodbstore.DynamoDBStorage{
    Table:               "CertMagic"
}
```

could you help me?

**Comment by MichaelMure** [comment]
See https://github.com/silinternational/certmagic-storage-dynamodb/pull/8

**Comment by danlsgiga** [comment]
Just had the same issue. Looking forward to see that PR merged ;)

**Comment by fillup** [comment]
Sorry all for the delay in merging the PR through to master. I just merged it through and released it as `2.0.0`  - https://github.com/silinternational/certmagic-storage-dynamodb/releases/tag/2.0.0
